dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/DefaultClients.java

```diff
@@ -67,6 +67,11 @@
         return DialogueFutures.transform(response, deserializer::deserialize);
     }
 
+    @Override
+    public <T> T callBlocking(EndpointChannel channel, Request request, Deserializer<T> deserializer) {
```

Comment:
@carterkozak / @CRogers for capturing the discussion: we're worried that if someone has the new version of dialogue-target, but old version of dialogue-serde, they could get a runtime failure.